src/common/actions/constant.js

```diff
@@ -0,0 +1,66 @@
+import STRINGS from "../localization";
+import constant from "../constant";
+
+export const THEMES = ["Default", "Light", "Dark"];
+
+export const FONT_SIZES = [
+  { key: "EXTRA_SMALL", title: `${STRINGS.extra_small}` },
+  { key: "SMALL", title: `${STRINGS.small_default}` },
+  { key: "MEDIUM", title: `${STRINGS.medium}` },
+  { key: "LARGE", title: `${STRINGS.large}` },
+  { key: "EXTRA_LARGE", title: `${STRINGS.extra_large}` },
+];
+export const FONT_FACES = [
+  { key: "AnmolLipiSG", title: `${STRINGS.anmol_lipi}` },
+  { key: "GurbaniAkharTrue", title: `${STRINGS.gurbani_akhar_default}` },
+  { key: "GurbaniAkharHeavyTrue", title: `${STRINGS.gurbani_akhar_heavy}` },
+  { key: "GurbaniAkharThickTrue", title: `${STRINGS.gurbani_akhar_think}` },
+];
+
+export const LANGUAGES = [
+  { key: "DEFAULT", title: `${STRINGS.default}` },
+  { key: "en-US", title: constant.ENGLISH },
+  { key: "es", title: constant.ESPANOL },
+  { key: "fr", title: constant.FRANCAIS },
+  { key: "it", title: constant.ITALIANO },
+  { key: "hi", title: constant.HINDI },
+  { key: "pa", title: constant.PUNJABI },
+];
+
+export const TRANSLITERATION_LANGUAGES = [
+  { key: "ENGLISH", title: `${STRINGS.english}` },
+  { key: "HINDI", title: STRINGS.hindi },
+  { key: "SHAHMUKHI", title: STRINGS.shahmukhi },
+  { key: "IPA", title: STRINGS.ipa },
```

Comment:
You should write a comment here explaining what is IPA or at least telling its full form as IPA doesn't sound like any language.